sensors/limelight.py

```diff
@@ -146,6 +146,9 @@ def set_led_mode(self, mode: config.limelight_led_mode) -> None:
 
     def get_led_mode(self) -> config.limelight_led_mode:
         return self.table.getNumber("ledMode", 0)
+    
+    def get_target_id(self) -> float:
+        return self.tid
```

Comment:
Can we refactor this to not be tid? tid is very hard to remember what it actually means.